modelbuilders_bench/mb_utils.py

```diff
@@ -23,7 +23,7 @@ def get_accuracy(true_labels, prediction):
     errors = 0
     for i, true_label in enumerate(true_labels):
         pred_label = 0
-        if isinstance(prediction[i], (float, np.single, np.float)):
+        if isinstance(prediction[i], (float, np.single)):
```

Comment:
It's better to specify more familiar type names
```suggestion
        if isinstance(prediction[i], (np.float32, np.float64)):
```